testdata/main.go

```diff
@@ -136,4 +137,6 @@ func main() {
 	b2.Write(nil)
 	rand.Read(nil)
 	mrand.Read(nil)
+
+	ioutil.ReadFile("main.go") // UNCHECKED
```

Comment:
What does UNCHECKED mean/what is this line testing? I don't see any change that actually makes the tests use a non-empty list of excludes.